cmd/id-exporter/main.go

```diff
@@ -267,17 +268,30 @@ func main() {
 	err = features.Set(cfg.ContactExporter.Features)
 	cmd.FailOnError(err, "Failed to set feature flags")
 
+	// Setup database client.
 	dbURL, err := cfg.ContactExporter.DB.URL()
 	cmd.FailOnError(err, "Couldn't load DB URL")
 
+	if !*useDefaultIsolationLevel {
+		conf, err := mysql.ParseDSN(dbURL)
+		cmd.FailOnError(err, "Couldn't parse DB URL as DSN")
+
+		// Transaction isolation level READ UNCOMMITTED trades consistency for
+		// performance.
+		conf.Params = map[string]string{
+			"tx_isolation": "'READ-UNCOMMITTED'",
+		}
```

Comment:
What if parsing the dbURL resulted in conf.Params already being populated? Don't want to overwrite the whole dict, and maybe don't even want to overwrite this one key if it's already been specified for some reason. Should add some checks to make sure you're not dropping anything on the floor here.

Reply:
Oh gosh, I totally forgot to change this to an assignment rather than a literal. Copypasta 🤦🏻‍♀️